attestation-service/verifier/src/lib.rs

```diff
@@ -152,6 +166,12 @@ pub trait Verifier {
         expected_report_data: &ReportData,
         expected_init_data_hash: &InitDataHash,
     ) -> Result<TeeEvidenceParsedClaim>;
+
+    async fn generate_challenge_extra_params(
```

Comment:
Do not use `extra_params` as function name here, because this is a conception of KBS protocol. I think `generate_challenge` is enough.

Reply:
fixed